snippets/nordic-bt-rpc/README.rst

```diff
@@ -1,11 +1,23 @@
-.. _nordic-rpc-host:
```

Comment:
To be on the safe side, keep the both doc labels. If we have earlier linked to this doc using :ref:`nordic-rpc-host` the link will be broken.

Reply:
I don't think we linked to this in many places. The ref itself was added 14h ago here: https://github.com/nrfconnect/sdk-nrf/pull/15634/files#diff-36440314ecd7741e2df0b84b51ba008dc0a72e83b56995def3c56979de7e4945

Reply:
It does no harm if we keep it there. Up to you.

Reply:
No there is no reference to the former link. I'll leave the new one only.